core/server/data/migrations/versions/4.46/2022-04-13-12-02-fill-created-at-newsletters.js

```diff
@@ -0,0 +1,18 @@
+const logging = require('@tryghost/logging');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Setting missing created_at values for existing newsletters');
+
+        const updatedRows = await knex('newsletters')
+            .where('created_at', null)
+            .update('created_at', new Date());
```

Comment:
I think this needs to be 

```
const currentTimestamp = knex.raw('CURRENT_TIMESTAMP');

...


    .update('created_at', currentTimestamp);
```

Reply:
I've updated it. I would expect Date objects to get mapped correctly, but I couldn't find any documentation about it